cmd/sonobuoy/app/delete.go

```diff
@@ -34,7 +35,7 @@ var deleteFlags struct {
 	wait       int
 }
 
-func init() {
+func NewCmdDelete(out io.Writer) *cobra.Command {
```

Comment:
You added the io.Writer to all of these functions, why? I absolutely get that at some point down the line you may want to mock out `io` for the commands but this doesn't plumb it through any farther so the variable is just unused everywhere.

Seems like for now this io stuff should be out of scope for this change. Let me know if there something I wasn't seeing w.r.t. to it though.